api/package.json

```diff
@@ -13,26 +13,12 @@
     "lint-fix-jessie": "eslint -c '.eslintrc-jessie.js' --fix '**/*.{js,jsx}'",
     "lint-check-jessie": "eslint -c '.eslintrc-jessie.js' '**/*.{js,jsx}'"
   },
-  "devDependencies": {
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-config-jessie": "^0.0.6",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-jsdoc": "^39.3.3",
-    "prettier": "^2.5.1"
-  },
   "dependencies": {
     "@endo/eventual-send": "^0.15.5",
     "@endo/marshal": "^0.6.9",
+    "@agoric/dapp-fungible-faucet-ui": "^0.0.1",
```

Comment:
The API having a dependency on the UI seems backwards

Reply:
I can try to flip this around, but it was this way when I arrived here.

Reply:
Yeah it's out of scope, but maybe we should have an issue to track